src/modules/storage/LedgerStorage.ts

```diff
@@ -233,7 +233,17 @@ export class LedgerStorage extends Storages
             vol_24h         BIGINT(20) UNSIGNED NOT NULL,
             change_24h      BIGINT(20),
             PRIMARY KEY (last_updated_at)
-        );
+            );
+       Create TABLE IF NOT EXISTS account (
+            address         TEXT        NOT NULL,
+            lock_bytes      INTEGER     NOT NULL,
+            txs_count       INTEGER     NOT NULL,
+            total_recieved  BIGINT(20)  UNSIGNED NOT NULL,
+            total_sent      BIGINT(20)  UNSIGNED NOT NULL,
+            total_reward    BIGINT(20)  UNSIGNED NOT NULL,
+            balance         BIGINT(20)  UNSIGNED NOT NULL,
+            PRIMARY KEY (address(64))
+            );
```

Comment:
```suggestion
       
       CREATE TABLE IF NOT EXISTS account (
            address                TEXT        NOT NULL,
            txs_count              INTEGER     NOT NULL,
            total_received_amount  BIGINT(20)  UNSIGNED NOT NULL,
            total_sent_amount      BIGINT(20)  UNSIGNED NOT NULL,
            total_reward_amount    BIGINT(20)  UNSIGNED NOT NULL,
            total_freeze_amount    BIGINT(20)  UNSIGNED NOT NULL,
            balance                BIGINT(20)  UNSIGNED NOT NULL,
            PRIMARY KEY (address(64))
            );
```
Please add the document here.
https://github.com/bosagora/stoa/blob/v0.x.x/docs/Database_Schema.md